lib/time.py

```diff
@@ -23,3 +100,31 @@ def sleep(secs):
 
 def time():
   return float(Now().UnixNano()) / Second
+
+
+def strftime(format, tt=None):  # pylint: disable=missing-docstring,redefined-builtin
+  t = (Unix(int(mktime(tt)), 0) if tt else Now()).Local()
+  ret = []
+  prev, n = 0, format.find('%', 0, -1)
+  while n != -1:
+    ret.append(format[prev:n])
+    next_ch = format[n + 1]
+    c = _strftime_directive_map.get(next_ch)
+    if c is NotImplementedError:
```

Comment:
You could use the builtin NotImplemented if you wanted to save a few characters.